src/nvme/ioctl.h

```diff
@@ -1789,6 +1789,39 @@ static inline int nvme_get_log_media_unit_stat(int fd, __u16 domid,
 }
 
 
+/**
+ * nvme_get_log_support_cap_config_list() -
+ * @fd:		File descriptor of nvme device
+ * @domid:	Domain Identifier selection, if supported
+ *
+ * Return: The nvme command status if a response was received (see
+ * &enum nvme_status_field) or -1 with errno set otherwise
+ */
+static inline int nvme_get_log_support_cap_config_list(int fd, __u16 domid,
+			struct nvme_supported_cap_config_list_log *cap)
+{
+	struct nvme_get_log_args args = {
+		.args_size = sizeof(args),
+		.fd = fd,
+		.result = NULL,
+		.timeout = NVME_DEFAULT_IOCTL_TIMEOUT,
+		.lid = NVME_LOG_LID_SUPPORTED_CAP_CONFIG_LIST,
+		.lpo = 0,
+		.log = cap,
+		.len = sizeof(*cap),
+		.nsid = NVME_NSID_NONE,
+		.csi = NVME_CSI_NVM,
+		.lsi = NVME_LOG_LSI_NONE,
+		.lsp = NVME_LOG_LSP_NONE,
+		.domid = domid,
+		.uuidx = NVME_UUID_NONE,
+		.rae = false,
+		.ot = false,
+	};
+	return nvme_get_log(&args);
+}
+
+
```

Comment:
Please avoid this double new line. I see there are already a few of those in this section. Please send a separate cleanup patch.

Reply:
@igaw 
There are many places in this section double new line is being used, taken that as reference. Will change it to single new line and give you separate patch

Reply:
I know the file is a bit inconsistent. So the best thing would do have just one separate cleanup patch.